docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: '3.8'
+
+services:
+  mysqldb:
+    platform: linux/arm64
+    image: mysql:8.4.0
+    restart: unless-stopped
+    env_file: ./.env
+    environment:
+      - MYSQL_ROOT_PASSWORD=$MYSQL_ROOT_PASSWORD
+      - MYSQL_DATABASE=$MYSQL_DATABASE
+    ports:
+      - $MYSQL_DOCKER_PORT:$MYSQL_LOCAL_PORT
```

Comment:
the first one value stated here corresponds to the local port that will be used to access the service outside the docker. The second one - port within the docker. So please consider it

Reply:
Ok, I fix it, please check my solution